### DISCUSSION TITLE
pageManager' does not exist in type 'EditorConfig'

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#4556] [General]
Hi, I'm just learning how to use grapesjs, I'm using version 0.19.5 and when I try to use the pageManager, i get this error ' **Object literal may only specify known properties, and 'pageManager' does not exist in type 'EditorConfig'.**'  
Am I doing something stupid (could be :) ) or there is another problem going on there? Have any of you guys met this before?
Thank's!

**Comment by artf** [comment]
No, nothing stupid, it simply missing in the declaration file. I will fix it in the next release, for now you have to ignore it.

**Comment by badiuciprian** [comment]
ok, thanks. I will follow the news
